src/awkward/_v2/_connect/rdataframe/from_rdataframe.py

```diff
@@ -0,0 +1,161 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import awkward as ak
+
+import awkward._v2._lookup  # noqa: E402
+import awkward._v2._connect.cling  # noqa: E402
+
+import ROOT
+import cppyy
+import ctypes
+from awkward._v2.types.numpytype import primitive_to_dtype
+
+cpp_type_of = {
+    "float64": "double",
+    "int64": "int64_t",
+    "complex128": "std::complex<double>",
+    "uint8": "uint8_t",
+}
+
+np = ak.nplike.NumpyMetadata.instance()
+numpy = ak.nplike.Numpy.instance()
+
+cppyy.add_include_path("src/awkward/_v2/_connect")
```

Comment:
Yes, I agree. I was thinking that we do not need to define the include path in the project code. It seems, there are better ways, since we are using cmake already. I found a reference to this repo: https://github.com/jclay/cppyy-knearestneighbors-example. It looks like with that auto-loading should work - and no need to deal with headers in our own code.

Reply:
done